src/genegraph/source/graphql/common/cache.clj

```diff
@@ -47,8 +47,8 @@
 (defn expire-resolver-cache-on-event! [event]
```

Comment:
Maybe use `clojure.core/run!` and it looks like this can be private?
```clojure
(defn- expire-resolver-cache-on-event!
  "Expire EVENT from the GraphQL resolver cache."
  [event]
  (when (running?)
    (rocksdb/rocks-delete-with-prefix! resolver-cache-db ::expire-always)
    (run! (partial rocksdb/rocks-delete-with-prefix! resolver-cache-db)
          (-> event ::q/model q/referenced-resources)))
  event)
```